src/services/ml5-service.ts

```diff
@@ -22,7 +23,7 @@ const classifyImage = async ({imageUrl}) => {
   log('classifying...');
   await requireMl5();
   const image = await loadImage(imageUrl);
-  const classifier = await window.ml5.imageClassifier('MobileNet');
+  const classifier = await window['ml5'].imageClassifier('MobileNet');
```

Comment:
optional: can global augmentation work here?

https://www.typescriptlang.org/docs/handbook/declaration-merging.html#global-augmentation

Reply:
I was trying to avoid global augmentation since it would be cumbersome for all the properties we wanted to add to window on the fly. Do you think it's the right approach?

Reply:
Okay to leave as-is.  File a github issue to determine if that's needed.

Reply:
Good idea. Created issue #3062